Neos.Neos/Classes/Command/NodeCommandControllerPlugin.php

```diff
@@ -66,6 +67,12 @@ class NodeCommandControllerPlugin implements EventDispatchingNodeCommandControll
      */
     protected $nodeDataRepository;
 
+    /**
```

Comment:
Do we have a guideline when we use the old Annotation and when we use the PHP Attribute?